post-install.yml

```diff
@@ -169,15 +171,70 @@
             production: https://{{ deploy_env }}-postgresapi.{{ domain_name }}
         dest: ~/postgresapi.yml
 
-    - name: Get service list
+    - name: Post Install Tasks | Get service list
       shell: >
         tsuru service-list
       register: service_list
-    - name: Register service
+    - name: Post Install Tasks | Register service
       shell: >
         crane create ~/postgresapi.yml
       when: "not 'postgresql' in service_list.stdout"
 
+    # Elasticsearch
+
+    - name: Post Install Tasks | Tsuru app-create elasticsearchapi
+      ignore_errors: yes
```

Comment:
Why ignore_errors? We don't want to continue if there is a failure.
I don't understand why we have more *ignore_errors: yes* above